TACC-Cloud/hazmapper
# task/WG-264: Hazmapper suffix fix

## Overview: ##

## PR Status: ##


## Related Jira tickets: ##

* [WG-264](https://tacc-main.atlassian.net/browse/WG-264)

## Summary of Changes: ##

- Before this PR, the .hazmapper extenion was being pushed out of the input box every time the "File name required" text was rendered from the Yup validation.
- Now the hazmapper extension will stay inside the input box when the validation text appears, making the UI look cleaner. 

## Testing Steps: ##
1. Run with local geoapi backend (master branch is fine)
2. Open the "Create Map" modal
3. Verify that the ".hazmapper" extension is present in the Custom File Name field
4. Start typing a file name in that field, then delete the text and click out/unfocus from the Custom File Name field so that the "File name is required" text pops up. 
5. When it does, verify that the ".hazmapper" extension text is still inside the input box 
6. Compare with the screenshots below

## UI Photos:
<img width="670" alt="Screenshot 2024-03-28 at 3 24 41 PM" src="https://github.com/TACC-Cloud/hazmapper/assets/50084480/c0b7cae7-7b18-4803-8402-0728e758582f">
<img width="670" alt="Screenshot 2024-03-28 at 3 25 20 PM" src="https://github.com/TACC-Cloud/hazmapper/assets/50084480/a0388289-aaaa-44f0-a1a9-3cb2b9c2b78c">

## Notes: ##
